packages/core/src/provider/plugins/swc.ts

```diff
@@ -79,6 +79,13 @@ export const pluginSwc = (): RsbuildPlugin => ({
 
       applyDecorator(swcConfig, config.output.enableLatestDecorators);
 
+      if (swcConfig.jsc?.externalHelpers) {
+        chain.resolve.alias.set(
+          '@swc/helpers',
+          path.dirname(require.resolve('@swc/helpers/package.json')),
```

Comment:
@swc/helpers is a peerDependencies of @rspack/core in 0.5, before the alias is added by rspack